### DISCUSSION TITLE
Converting object to an encodable object failed

### DISCUSSION THREAD

**Discussion by piyastk** [supabase/supabase#25919] [Show and tell]
flutter: Converting object to an encodable object failed: _Map len:1. 

while i am submitting json , i am getting this error. any solution on this ?)

**Comment by dshukertjr** [comment]
Could you provide a sample code that you ran that got you this error that you are facing? My guess is that you tried to pass a JSON encoded string to the .insert() method, but this is not necessary, and you should just pass a Map to the method in order to insert or update data.
